### PR TITLE
Update vpn-profile-options.md

### PR DESCRIPTION
**Pull request by msarcletti**
Adding additional information for the scope / limitation of the VPN proxy settings configuration

### CODE REVIEW

File: windows/security/identity-protection/vpn/vpn-profile-options.md [hunk 1]
```diff
@@ -45,6 +44,9 @@ The following table lists the VPN settings and whether the setting can be config
 | LockDown | no |
 | Windows Information Protection (WIP) | yes |
 | Traffic filters | yes |
+| Proxy settings | yes, by PAC/WPAD file or server and port |
+>[!NOTE] 
+>VPN proxy settings are only used on Force Tunnel Connections. On Split Tunnel Connections the general proxy settings are used.
```
Comment (illfated) [inline windows/security/identity-protection/vpn/vpn-profile-options.md:49]: May I request some spacing between the table and the Note blob, as well as a MarkDown indent marker compatibility spacing?
```suggestion
| Proxy settings | yes, by PAC/WPAD file or server and port |

> [!NOTE] 
> VPN proxy settings are only used on Force Tunnel Connections. On Split Tunnel Connections the general proxy settings are used.
```

Comment (msarcletti) [reply windows/security/identity-protection/vpn/vpn-profile-options.md:49]: Thanks, formating suggestions are always very welcome

File: windows/security/identity-protection/vpn/vpn-profile-options.md [hunk 2]
```diff
@@ -45,6 +44,10 @@ The following table lists the VPN settings and whether the setting can be config
 | LockDown | no |
 | Windows Information Protection (WIP) | yes |
 | Traffic filters | yes |
+| Proxy settings | yes, by PAC/WPAD file or server and port |
+
+> [!NOTE] 
+> VPN proxy settings are only used on Force Tunnel Connections. On Split Tunnel Connections the general proxy settings are used.
```
Comment (illfated) [inline windows/security/identity-protection/vpn/vpn-profile-options.md:50]: Might be a cosmetic detail, but it feels to me as if there should be a readability comma here:
```suggestion
> VPN proxy settings are only used on Force Tunnel Connections. On Split Tunnel Connections, the general proxy settings are used.
```

**Review by illfated — APPROVED** [review]
#sign-off

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Pull request merged**